app/com/linkedin/drelephant/schedulers/AirflowScheduler.java

```diff
@@ -58,10 +62,14 @@ public AirflowScheduler(String appId, Properties properties, SchedulerConfigurat
   }
 
   private void loadInfo(String appId, Properties properties) {
-    // Update the 4 Ids
+    // examples:
+    // my_amazing_task_id
     _taskId = properties.getProperty(AIRFLOW_TASK_ID);
+    // 2016-06-27T01:30:00
```

Comment:
That's why `getFlowExecId` returns `${_dagId}/${_dagRunExecutionDate}` and `getJobExecId` returns `${_dagId}/${_taskId}/${_taskInstanceExecutionDate}`. Essentially, those are the primary keys for a `dag run`/`task instance` in Airflow which correspond to `flow execution id`/`job execution id` in Azkaban. The terminology is a bit different but I think I am constructing the IDs correctly. Let me know if you think you are/I am still confused about something.